posts/models.py

```diff
@@ -6,3 +6,4 @@
 class Post(models.Model):
     content = models.CharField(max_length=256)
     author = models.ForeignKey(User)
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Seems like I have forgotten this part in my other pull request.
Could you be so kind and add `updated_at` aswell?